README.rst

```diff
@@ -69,6 +69,12 @@ Enable auto capture event logs in your nameko configuration file:
     EVENTLOG_DISPATCHER:
       auto_capture: true
       entrypoints_to_exclude: []
+      event_types:
+        generic: log_event
+        entrypoint_fired: entrypoint_fired
```

Comment:
I think the config structure might get a bit confusing here. Part of the problem is probably that we are overloading the term `event_type`. Maybe we should rename the `event_type` in the payload to just `event` and change the config to something like:
```
EVENTLOG_DISPATCHER:
  auto_capture: true
  entrypoints_to_exclude: []
  event_type: log_event
  entrypoint_fired_event: entrypoint_fired
```